src/main/java/com/example/App.java

```diff
@@ -5,6 +5,10 @@
 public class App {
     public App() throws IOException {
 //        super();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://docker.for.mac.localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)